### PR TITLE
[4.0] codestyle

### PR DESCRIPTION
**Pull request by brianteeman**
do not use units when the value is 0 eg 0 not 0px
always have a space after the colon eg color: $red not color:$red

### CODE REVIEW

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 6a1d2c3b512e85bca7309c90b9102162c3367474<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28047">issues.joomla.org/tracker/joomla-cms/28047</a>.</sub>

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 6a1d2c3b512e85bca7309c90b9102162c3367474<br /><br />Code review. There are also in addition other, e.g. sidebar_nav.scss, toolbar.scss etc. (total 8) changed. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28047">issues.joomla.org/tracker/joomla-cms/28047</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28047">issues.joomla.org/tracker/joomla-cms/28047</a>.</sub>

File: administrator/templates/atum/scss/blocks/_sidebar-nav.scss [hunk 1]
```diff
@@ -37,13 +37,13 @@
...
 
       &:before
       {
-        position:absolute;
-        left:0;
-        top:0.75rem;
+        position: absolute;
+        left: 0;
+        top: 0.75rem;
         border-right: 1px solid  var(--atum-link-color);
```
Comment (Quy) [inline administrator/templates/atum/scss/blocks/_sidebar-nav.scss]:
```suggestion
        border-right: 1px solid var(--atum-link-color);
```

File: administrator/templates/atum/scss/blocks/_sidebar.scss [hunk 2]
```diff
@@ -150,7 +150,7 @@
 
           // invert colors
           background-color:  var(--atum-sidebar-bg);
```
Comment (Quy) [inline administrator/templates/atum/scss/blocks/_sidebar.scss]:
```suggestion
          background-color: var(--atum-sidebar-bg);
```

File: administrator/templates/atum/scss/vendor/bootstrap/_buttons.scss [hunk 3]
```diff
@@ -7,7 +7,7 @@
   background-color: var(--atum-link-color);
   color: var(--atum-text-light);
   border-color:  var(--atum-link-color);
```
Comment (Quy) [inline administrator/templates/atum/scss/vendor/bootstrap/_buttons.scss]:
```suggestion
  border-color: var(--atum-link-color);
```

File: administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss [hunk 4]
```diff
@@ -59,7 +59,7 @@ joomla-tab {
         border-radius: 0;
         box-shadow: none;
         margin: -1px;
-        border:1px solid  var(--bluegray);
+        border: 1px solid  var(--bluegray);
```
Comment (Quy) [inline administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss]:
```suggestion
        border: 1px solid var(--bluegray);
```

File: administrator/templates/atum/scss/blocks/_sidebar-nav.scss [hunk 5]
```diff
@@ -37,13 +37,13 @@
       padding:  0.35rem 0.35rem 0.35rem 1.2rem;
```
Comment (Quy) [inline administrator/templates/atum/scss/blocks/_sidebar-nav.scss]:
```suggestion
      padding: 0.35rem 0.35rem 0.35rem 1.2rem;
```

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**